src/documents/activities/StatesandContextsMapping.html.md.handlebars

```diff
@@ -0,0 +1,42 @@
+---
+title: States and Contexts Mapping
+layout: content
+category: Activities
+---
+
+This activity breaks down the User States and Contexts tool into modular pieces and actionable steps, and it is also a re-thinking of the original User States and Contexts tool.
+
+Here, states are reframed as functional requirements. Deficit language or disability classifications that are functionally ambiguous are avoided.
+
+Placing needs that are commonly met in the middle and needs that are not well met further out, demonstrates that edge needs are greater in number, and communicates the phenomena that by meeting edge needs, you also meet the needs in the middle.
+
+In this activity, the states and contexts facets can be swapped in and out depending on the context and the design challenge. Facets can also be broken down into more specific considerations. For example, cognitive access might include memory, focus and other considerations. 
+
+This activity is intended to be used in the context of co-design where a diverse set of co-designers, including people who presently have difficulty using the design, are active participants in the design process. Co-designers are able to create their own facets if they can't find a desired state or context within the provided set.
+
+<ul>
+<li>The exercise is about <strong>stretching and reaching</strong>: to include people who are currently at the edges and who are therefore usually excluded from the design process.</li>
+<li>It is also about <strong>balance and flexibility</strong>: creating a design that does not compromise the experience of one person to make room for the requirements of another.</li>
+</ul> 
+
+## Step 1
+Choose the facets that are relevant to the needs, preferences and the context you are designing for or critiquing.
+![A diagram showing individual state and context facets](/images/user states & contexts-modular-01.png)
```

Comment:
Images in markdown files need a bit more semantics. We manually add HTML to properly describe images. For example, see: https://github.com/fluid-project/docs-inclusive-learning/blob/84c7b6077b04701378f23370d74d6e9672601f99/src/documents/Metadata.html.md.handlebars#L28

The markup in that link looks a bit convoluted because of the multiple brackets (one is for the thumbnail, another for the image, and another for the actual link to the larger image).